CMakeLists.txt

```diff
@@ -15,12 +15,10 @@ option( EXIV2_ENABLE_PNG              "Build with png support (requires libz)"
 cmake_dependent_option(EXIV2_ENABLE_NLS "Build native language support (requires gettext)" ON "UNIX" OFF)
 option( EXIV2_ENABLE_PRINTUCS2        "Build with Printucs2"                                  ON  )
 option( EXIV2_ENABLE_LENSDATA         "Build including lens data"                             ON  )
-option( EXIV2_ENABLE_VIDEO            "Build video support into library"                      OFF )
```

Comment:
There are two more occurrences of that variable in the code. Could you take care of removing them?

```
cmake/printSummary.cmake
49:OptionOutput( "Building video support:             " EXIV2_ENABLE_VIDEO              )

cmake/packaging.cmake
56:if ( EXIV2_ENABLE_VIDEO )
```